doc/api/util.md

```diff
@@ -1989,6 +1989,38 @@ const channel = new MessageChannel();
 channel.port2.postMessage(signal, [signal]);
 ```
 
+## `util.aborted(signal, resource)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `signal` {AbortSignal}
+* `resource` {any} Any non-null entity
```

Comment:
This really isn't `any`, correct? It needs to be an object

Reply:
Also, it should be stressed here that the reference to `resource` is held weakly.

Reply:
In the validator I have allowed objects, arrays, functions 
so make the time {Object|Array|Function}

Reply:
updated could you check